scoped_model_counter/lib/main.dart

```diff
@@ -0,0 +1,80 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:scoped_model/scoped_model.dart';
+
+void main() {
+  // Initialize the model. Can be done outside a widget, like here.
+  var counter = Counter();
+
+  // We can now interact with the model from outside of the Flutter widget tree.
+  // Here, we wait 5 seconds and increment the counter.
+  Timer(
+    const Duration(seconds: 5),
+    () => counter.increment(),
+  );
```

Comment:
Maybe this could be setup as a repeated interaction that occurs every 5 sec?

Reply:
Done.